paris/src/main/java/com/airbnb/paris/typed_array_wrappers/MapTypedArrayWrapper.kt

```diff
@@ -104,6 +106,11 @@ internal class MapTypedArrayWrapper constructor(
     override fun getFloat(index: Int): Float =
         getValue(index, { resId -> resources.getFloat(resId) })
 
+    override fun getFont(index: Int): Typeface? =
+        getFontValue(index,
+            { resId -> if (isNullRes(resId)) null else ResourcesCompat.getFont(context, resId) },
+            { stringValue -> Typeface.create(stringValue, Typeface.NORMAL) })
```

Comment:
Would it make sense to check `stringValue` against a predetermined list of possible values?

Reply:
Given that the list of system font families is loaded dynamically from configuration file (see `TextView.setTypefaceFromAttrs(...)`, `Typeface.create(String,int)` and `Typeface.init()`), I guess it would be hard to determine and maintain such list across different Android versions and devices.